src/libraries/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj

```diff
@@ -3,8 +3,6 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461-windows</TargetFrameworks>
     <IsPackable>true</IsPackable>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <ServicingVersion>1</ServicingVersion>
```

Comment:
Shouldn't we leave ServicingVersion set?

Reply:
Yes I believe we should since I see this version on nuget.org: https://www.nuget.org/packages/System.Security.Cryptography.Xml/6.0.1

Reply:
Oh, right. Thanks for noticing, @ericstj and @bartonjs.

Reply:
Fixed.